azurerm/internal/services/compute/tests/windows_virtual_machine_resource_scaling_test.go

```diff
@@ -72,6 +72,36 @@ func TestAccWindowsVirtualMachine_scalingDedicatedHost(t *testing.T) {
 	})
 }
 
+func TestAccWindowsVirtualMachine_scalingDedicatedHostUpdate(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_windows_virtual_machine", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: checkWindowsVirtualMachineIsDestroyed,
+		Steps: []resource.TestStep{
+			{
+				Config: testWindowsVirtualMachine_scalingDedicatedHost(data),
+				Check: resource.ComposeTestCheckFunc(
+					checkWindowsVirtualMachineExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(
+				"admin_password",
+			),
+			{
+				Config: testWindowsVirtualMachine_scalingDedicatedHostUpdate(data),
+				Check: resource.ComposeTestCheckFunc(
+					checkWindowsVirtualMachineExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(
+				"admin_password",
+			),
```

Comment:
since we have support for removing the Dedicated Host ID, can we add a test for starting without a dedicated host, then using one and then not using a dedicated host?

Reply:
Test added